pkgs/tools/X11/virtualgl/lib.nix

```diff
@@ -19,6 +19,12 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  postPatch = ''
+    # script calls exec, which fails with plain sh
+    substituteInPlace ./server/vglrun.in \
+      --replace '#!/bin/sh' '#!${stdenv.shell}'
```

Comment:
Can this be also fixed by the `patchShebangs` script?

Reply:
That won't give me a bash but the path to a `sh` (I tried).